packages/xchain-ethereum/src/client.ts

```diff
@@ -345,7 +343,7 @@ export default class Client implements XChainClient, EthereumClient {
         account.tokens.forEach((token) => {
           const decimals = parseInt(token.tokenInfo.decimals)
           const tokenAsset = assetFromString(`${AssetETH.chain}.${token.tokenInfo.symbol}-${token.tokenInfo.address}`)
-          if (tokenAsset) {
+          if (tokenAsset && getTokenAddress(tokenAsset)) {
```

Comment:
What about the case if `token.tokenInfo.symbol` is `undefined`. As [mentioned before](https://github.com/xchainjs/xchainjs-lib/pull/171/commits/3ebe56a2808e5d9528650c1c3b187042d2d2cf36#r554060657), pls validate `symbol`.

Steps are:
1. Validate `symbol`
2. Validate `address`
3. Use valid values in `assetToString`